### PR TITLE
Fix recipes for sonata-project/media-orm-pack

### PR DESCRIPTION
**Pull request by covex-nn**
| Q             | A
| ------------- | ---
| License       | MIT

This PR will allow to update database schema after installation `sonata-project/classification-bundle` with new recipe from #389

### CODE REVIEW

File: sonata-project/media-orm-pack/1.0/post-install.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<bg=blue;fg=white> Integration SonataMediaBundle with SonataClassificationBundle </>
+
+  * If you are using <info>sonata-project/classification-bundle</> fix annotations for
```
Comment (kunicmarko20) [inline sonata-project/media-orm-pack/1.0/post-install.txt]: I think you need a `,` after the bundle name. What does our spellcheck say, summoning @greg0ire 😄

File: sonata-project/media-orm-pack/1.0/post-install.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<bg=blue;fg=white> Integration SonataMediaBundle with SonataClassificationBundle </>
```
[the remark this answers is not in the document]
Comment (covex-nn) [reply sonata-project/media-orm-pack/1.0/post-install.txt]: You a right )

**Comment by covex-nn** [comment]
I guess we can merge it. If there any errors in `post-install.txt`, i will fix them with another PR

**Comment by covex-nn** [comment]
Magic did not happen) @kunicmarko20 may be PR should be approved again? i pushed two commits after the last approve

**Comment by covex-nn** [comment]
@symfony-flex-server review please

**Pull request merged**